src/components/Ffuf/FfufTool.tsx

```diff
@@ -7,7 +7,10 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "Fuzz Faster U Fool (ffuf)";
+// Component constants.
```

Comment:
Component constants need to be placed within the function, i.e. the FfufTool function.